disco/pydss/pydss_configuration_base.py

```diff
@@ -26,7 +26,17 @@
         "controller_type": ControllerType.PV_CONTROLLER.value,
         "name": "volt_var_1",
         "filename": DEFAULT_CONTROLLER_CONFIG_FILE
-    }
+    },
+    {
+        "controller_type": ControllerType.PV_CONTROLLER.value,
+        "name": "volt_var_disco",
```

Comment:
@jgu2 I added your change to this PR even though it is duplicate so that I could ensure that the controller gets registered with PyDSS.